queries/foam/injections.scm

```diff
@@ -0,0 +1,9 @@
+;; Pass code blocks to Cpp highlighter
+(code (code_body) @cpp)
+
+;; Pass identifiers to Go highlighter (Cheating I know)
+;;((identifier) @lua)
+
+;; Highlight regex syntax inside literal strings
+((string_literal) @regex)
```

Comment:
Are all strings in your language regexes ??

Reply:
Kind of, yes, we're passing all strings to POSIX regex engines internally so I thought why not highlight it as such. Non-regex words are parsed as identifiers.